Glue code to get content info metrics

hypest opened the pull request:
Expands #2380 to include the Android side of the call to get content information.

To test:
Use the WPAndroid PR to test: https://github.com/wordpress-mobile/WordPress-Android/pull/12191
 
PR submission checklist:

- [x] I have considered adding unit tests where possible.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

SergioEstevao commented:
@hypest any possibility to add a way to check the information directly in the demo app?

hypest commented:
> any possibility to add a way to check the information directly in the demo app?

We could add a menu item on the RN Debug menu, similar to the "Show html" we have added in the past, but that's not a scalable solution. We needed the "Show html" one for the UI tests so we added it, but this one is probably not as important to have.

Unlike the iOS demo app, the Android demo app doesn't have chrome/toolbar around the editor.
